app/jobs/migrate_files_to_valkyrie_job.rb

```diff
@@ -2,6 +2,10 @@
 # Responsible for conditionally enqueuing the file and thumbnail migration
 # logic of an ActiveFedora object.
 class MigrateFilesToValkyrieJob < Hyrax::ApplicationJob
+  # Define a logger for this job
+  def logger
+    @logger ||= Logger.new(Rails.root.join('log', 'migrate_files_to_valkyrie_job.log'))
```

Comment:
```suggestion
    @logger ||= Logger.new(Rails.root.join('tmp', 'imports', 'migrate_files_to_valkyrie_job.log'))
```

this prevents the log from being deleted when the pod ends